docs/source/eFeatures.rst

```diff
@@ -1787,3 +1787,16 @@ Difference between minimum and steady state during stimulation
 
     diff_min_duringstim: min_duringstim - steady_state_voltage_stimend
 
+Python efeature: spikes_per_burst
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Number of spikes in each burst.
```

Comment:
Could you also be explicit in the description that The first spike is ignored by default. This can be changed by setting ignore_first_ISI to 0.